.codecov.yml

```diff
@@ -29,21 +29,33 @@ flags:
     carryforward: true
     paths:
       - "x/"
-      - "!x/**/client/"
+      - "!x/**/client/**/*"
   client:
     carryforward: true
     paths:
-      - "x/**/client/"
+      - "x/**/client/**/*"
 
 ignore:
+  - "x/**/client/cli/query.go"
+  - "x/**/client/cli/query_params.go"
+  - "x/**/client/cli/tx.go"
+  - "x/**/client/cli/tx_*.go"
+  - "x/**/keeper/keeper.go"
+  - "x/**/keeper/msg_server.go"
+  - "x/**/keeper/grpc_query_params.go"
   - "x/**/types/codec.go"
   - "x/**/types/errors.go"
   - "x/**/types/keys.go"
+  - "x/**/types/key_*.go"
+  - "x/**/types/types.go"
+  - "x/**/types/genesis.go"
```

Comment:
Is it a mistake?

We should get 100% coverage for `genesis.go`

Reply:
Oh, sorry, yes, It was a mistake. I will remove the `genesis.go`